assert/assertions.go

```diff
@@ -2010,15 +2010,30 @@ func Eventually(t TestingT, condition func() bool, waitFor time.Duration, tick t
 		h.Helper()
 	}
 
-	ch := make(chan bool, 1)
-	checkCond := func() { ch <- condition() }
+	const failed = 0
+	const stop = 1
+	const noStop = 2
```

Comment:
```suggestion
	const (
		failed = iota
		stop
		noStop
	)
```

Though better names might be "checkFailed", "conditionPassed", and "conditionFailed".